### PR TITLE
backend/venv/activation

### PR DESCRIPTION
**Pull request by danielalbinoms**

### CODE REVIEW

**Review by alnavarroh — APPROVED** [review]

**Review by Eduardo-93 — APPROVED** [review]
lgtm👌

File: games_project/backend/public/import_text.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                wordOfDay = randomWord
+                break
+            elif accent[index] not in randomWord:
+                index += 1
+            else:
+                randomWord = choice(self.fileDictionary())
+                index = 0
+        return wordOfDay
+    
+#wordOfTheDay = WordOfTheDay('./palabras_rae.txt')
```
Comment (Darova93) [inline games_project/backend/public/import_text.py]: Please remove these comments

**Review by Darova93 — CHANGES_REQUESTED** [review]

File: games_project/backend/wordle.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+#        playerGuess.pop()
+#        if playerGuess in threeLetterWords.fileDictionary()
+#            return True
+#    elif playerGuess[-1] == "s":
+#        fourLetterWords = WordOfTheDay("./public/04.txt")
+#        playerGuess.pop()
+#        if playerGuess in fourLetterWords.fileDictionary()
+#            return True
+#    else:
+#        if playerGuess in fiveLetterWords.fileDictionary()
```
Comment (Darova93) [inline games_project/backend/wordle.py]: Remove this

**Review by Darova93 — CHANGES_REQUESTED** [review]
Remove 03.txt and 04.txt file

File: games_project/backend/wordle.py (hunk 2 above)
Comment (Eduardo-93) [reply games_project/backend/wordle.py]: Done

**Review by Darova93 — APPROVED** [review]

**Review by Bydel93 — APPROVED** [review]
LGTM

**Review by Bydel93 — APPROVED** [review]

**Pull request merged**